qiskit_acqua/operator.py

```diff
@@ -1383,3 +1384,88 @@ def _check_representation(self, targeted_represnetation):
             raise ValueError(
                 '"targeted_represnetation" should be one of "paulis", "grouped_paulis" and "matrix".'
             )
+
+    @staticmethod
+    def row_echelon_F2(matrix_in):
+        """
+        Computes the row Echelon form of a binary matrix on the binary
+        finite field
+
+        Args:
+            matrix_in (np.array): binary matrix
+
+        Returns:
+            matrix_out: matrix_in in Echelon row form
```

Comment:
Do you mind changing to the format:

Returns:
    np.ndarray: matrix_in in Echelon row form.

Reply:
same changes might need for other functions